circle.yml

```diff
@@ -151,6 +151,49 @@ commands:
           name: Restore all node_modules to proper workspace folders
           command: node scripts/circle-cache.js --action unpack
 
+  restore_cached_system_tests_deps:
```

Comment:
Sure. Do you know if the `description` is actually used anywhere? It's part of the circle.yml spec but IDK where it ends up, I think it may be the equivalent of a code comment: https://circleci.com/docs/2.0/configuration-reference/#commands-requires-version-21

Reply:
b6aaa99e6164c39e46dd0a62ada3f8dc6e8db681